app/messaging/back-office/process-backoffice.js

```diff
@@ -1,14 +1,15 @@
-const { getAll, getApplicationCount } = require('../../repositories/application-repository')
+const { searchApplications } = require('../../repositories/application-repository')
 const { backOfficeResponseMsgType, backOfficeResponseQueue } = require('../../config')
 const sendMessage = require('../../messaging/send-message')
-
+const util = require('util')
 const processBackOfficeRequest = async (msg) => {
   const responseMessage = msg.body
   const { sessionId } = msg
   try {
-    // Search application with SBI number and return requested page for selected offset and limit.
-    const applications = await getAll(msg.body.search.text, msg.body.offset, msg.body.limit)
-    const total = await getApplicationCount(msg.body.search.text)
+    console.log('search request received:', util.inspect(msg.body, false, null, true))
+    // Search application with SBI number/Reference/Status and return requested page for selected offset and limit.
```

Comment:
It would be better to move the comment into JSDoc on `searchApplications` then all callers would have access to the information.